### ISSUE TITLE
Evilcube desynchronization

### ISSUE THREAD

**Issue by rhythm-on-github** [Team-Cubey/cubey-issues#7]
Sometimes, evilcubes can get desyncronized for no apparent reason.
The circumstances that were happening while i found this were:
-multiple evilcubes (in this case 5 evilcubes)
-all on paths of the same length (in this case 5 cubes long)
-same direction path
-all right next to eachother
-all loaded at the same time

They slowly desynchronized (around 0.05 cubes for every time they changed direction)

**Comment by Hubza** [comment]
this is most likely delayed loading now i think about it. i'll have to figure out a way to stop all objects existing until the level is fully loaded.

**Comment by rhythm-on-github** [comment]
But that wouldn't explain how they keep desynchronizing more and more as time goes on though?

**Comment by Hubza** [comment]
oh that's true. thats really interesting. any clue what level this happens in?

**Comment by rhythm-on-github** [comment]
yes, it happens in precision pack level 5, at the bottom of the level

**Comment by Hubza** [comment]
this is really interesting, i cant exactly figure out why its happening. they all move at speed 3 and reverse at the same point

**Comment by rhythm-on-github** [comment]
damn wtf xd
